getting_started/step_by_step/scripting_first_script.rst

```diff
@@ -343,3 +399,40 @@ Here is the complete ``sprite_2d.gd`` file for reference.
             Position += velocity * (float)delta;
         }
     }
+ 
+ .. code-tab:: cpp C++
+
+    #ifndef MY_SPRITE_2D_H
+    #define MY_SPRITE_2D_H
+
+    #include <godot_cpp/variant/utility_functions.hpp>
+    #include <godot_cpp/classes/sprite2d.hpp>
+    #include <godot_cpp/core/math.hpp>
+
+    namespace CustomNamespace {
+
+    class MySprite2D : public Sprite2D {
+        GDCLASS(MySprite2D, godot::Object)
+
+        int speed = 400;
+        float angular_speed = Math_PI;
+
+    protected:
+        static void _bind_methods() {}
+
+    public:
+        MySprite2D() {
+            godot::UtilityFunctions::print("Hello, world!");
+        }
+
+        void _process(const double p_delta) override {
+            set_rotation(get_rotation() + (angular_speed * delta));
```

Comment:
```suggestion
            set_rotation(get_rotation() + angular_speed * p_delta);
```
Unnecessary and hurts readability IMO when adding parentheses without being needed for precedence 

Also correct variable name

Reply:
Will remove the parentheses agreed those do nothing. p_delta seems like the idomatic way to do it in C++ in both godot source code and GDExtension source code so I think it's best to not just copy the other scripting languages for that. Don't mind changing it though if you disagree.

Reply:
You are using `delta` not `p_delta`, you've got the suggestion backwards

Reply:
Oops I see now it seems like I forgot my brain this morning